sensor-value.html

```diff
@@ -111,6 +122,31 @@
 		padding-top: 2%;
 		width: 100%;
 	}
+  @media screen and (max-width: 760px) {
+    :host{
+      font-size: var(--sensor-value-font-size-min, 1.7em);
+    }
+		#value p {
+      line-height: var(--sensor-value-line-height-min, 0em);
```

Comment:
by default there was no line-height, so it doesn't need default values, please remove it